parser.hpp

```diff
@@ -242,29 +248,27 @@ namespace Sass {
     String* parse_interpolated_chunk(Token, bool constant = false);
     String* parse_string();
     String_Constant* parse_static_expression();
+    // String_Constant* parse_static_property();
     String_Constant* parse_static_value();
     String* parse_ie_property();
     String* parse_ie_keyword_arg();
     String_Schema* parse_value_schema(const char* stop);
-    Expression* parse_operators(Expression* factor);
     String* parse_identifier_schema();
     // String_Schema* parse_url_schema();
     If* parse_if_directive(bool else_if = false);
     For* parse_for_directive();
     Each* parse_each_directive();
     While* parse_while_directive();
+    Return* parse_return_directive();
+    Content* parse_content_directive();
+    void parse_charset_directive();
     Media_Block* parse_media_block();
     List* parse_media_queries();
     Media_Query* parse_media_query();
     Media_Query_Expression* parse_media_expression();
-    Feature_Block* parse_feature_block();
-    Feature_Query* parse_feature_queries();
-    Feature_Query_Condition* parse_feature_query();
-    Feature_Query_Condition* parse_feature_query_in_parens();
-    Feature_Query_Condition* parse_supports_negation();
```

Comment:
I don't like that this has been made so shallow for a couple a reasons.

- the heavy branching makes this code hard to read
- the current implementation is [modelled after the grammar](http://www.w3.org/TR/css3-conditional/#at-supports) so it's easy to spot inconsistencies.

I'd opt to backing out this change.